src/libraries/authUtils/tokenHasExpired.ts

```diff
@@ -0,0 +1,13 @@
+import jwtDecode from "jwt-decode";
+
+export interface JwtTokenModel {
+  exp: number;
+  sub: string;
+  auth: string;
+}
+
+export const tokenHasExpired = (token: any): boolean => {
+  const { exp } = jwtDecode<JwtTokenModel>(token);
+  const expirationTime = exp * 1000 - 60000;
```

Comment:
I would move this in a constant

Reply:
@SteveGT96 can you do that?

Move "60000" to a constant `TOKEN_EXPIRATION_TIMEOUT` in `src\consts.ts`

Reply:
@alefalezza It's done !